core/templates/components/button-directives/social-buttons.component.ts

```diff
@@ -18,7 +18,7 @@
 
 import { Component, OnInit } from '@angular/core';
 
-import { downgradeComponent } from '@angular/upgrade/static';
+
```

Comment:
Please d a self-review of all the changes, I see extra newlines in lots of files? (I've only reviewed top 5-6 files and I see this pattern)

Reply:
Done.